index-handlers/src/test/java/no/sikt/nva/nvi/index/SearchNviCandidatesHandlerTest.java

```diff
@@ -213,6 +226,30 @@ void shouldReturnPaginatedSearchResultWithAggregationTypeInIdIfGiven() throws IO
         assertThat(actualId, containsString(QUERY_AGGREGATION_TYPE + "=" + aggregationType));
     }
 
+    @Test
+    void shouldReturnPaginatedSearchResultWithOrderByInIdIfGiven() throws IOException {
+        mockOpenSearchClient();
+        var orderByValue = OrderByFields.CREATED_DATE.getValue();
+        var request = createRequest(TOP_LEVEL_CRISTIN_ORG, Map.of(QUERY_PARAM_ORDER_BY, orderByValue));
+        handler.handleRequest(request, output, context);
+        var response = GatewayResponse.fromOutputStream(output, PaginatedSearchResult.class);
+        var paginatedSearchResult = response.getBodyObject(PaginatedSearchResult.class);
+        var actualId = paginatedSearchResult.getId().toString();
+        assertThat(actualId, containsString(QUERY_PARAM_ORDER_BY + "=" + orderByValue));
+    }
+
+    @Test
+    void shouldReturnPaginatedSearchResultWithSortOrderInIdIfGiven() throws IOException {
+        mockOpenSearchClient();
+        var sortOrderValue = "desc";
+        var request = createRequest(TOP_LEVEL_CRISTIN_ORG, Map.of(QUERY_PARAM_SORT_ORDER, sortOrderValue));
+        handler.handleRequest(request, output, context);
+        var response = GatewayResponse.fromOutputStream(output, PaginatedSearchResult.class);
+        var paginatedSearchResult = response.getBodyObject(PaginatedSearchResult.class);
+        var actualId = paginatedSearchResult.getId().toString();
+        assertThat(actualId, containsString(QUERY_PARAM_SORT_ORDER + "=" + sortOrderValue));
```

Comment:
You have over 10 assertions where you do the same assertion ->  (queryParam=someValue). It seems like it would be convenient to implement something like:
```
    private static Matcher<String> containsQueryParam(String queryParam, String... values) {
        return containsString(queryParam + "=" + String.join(",", values));
    }
```